src/language_gdnative.c

```diff
@@ -351,6 +351,51 @@ static godot_pluginscript_language_desc lps_language_desc = {
 	},
 };
 
+static godot_pluginscript_language_desc lps_yue_language_desc = {
```

Comment:
I made a specific `lps_yue_language_desc` for custom names, extension and reserved keywords.
Turns out you can just register multiple language in one GDNative binding! Super cool :)

Since all of the functions are the same, it means changes / improvements on the Lua side will automatically be available for YueScript.

Reply:
> Turns out you can just register multiple language in one GDNative binding!

Yeah! Not only that, if we had any NativeScript classes or anything else we could add in the same library as well =]